bin/brew

```diff
@@ -44,4 +44,17 @@ fi
 
 HOMEBREW_LIBRARY="$HOMEBREW_REPOSITORY/Library"
 
-source "$HOMEBREW_LIBRARY/Homebrew/brew.sh"
+if [[ -n "$HOMEBREW_ENV_FILTERING" ]]
+then
+  PATH="/usr/bin:/bin:/usr/sbin:/sbin"
```

Comment:
@MikeMcQuaid Sure, if we're concerned about that vector. No objection to getting this working first.

What would you suggest for stdenv and `env :userpaths` (and possibly other things affected which I don't realize at the moment) though? Death sentence to all of them? Persist original PATH in `ORIG_PATH` and optionally restore it later? Something else?